modules.json

```diff
@@ -570,28 +570,18 @@
 			"usprop": "blockinfo|groups|editcount|registration|emailable|gender",
 			"format": "json"
         }
-    },
-    {
-        "filename": "get_contributors",
-        "docstring": "Demo of `Contributors` module: Get request to list all logged-in contributors and count of anonymous contributors to a page.",
-        "endpoint": "https://en.wikipedia.org/w/api.php",
-        "params": {
-            "action": "query",
-            "titles": "MediaWiki",
-            "prop": "contributors",
-            "format": "json"
-        }
-    },
+	},
 	{
-        "filename": "get_alldeletedrevs",
-        "docstring": "Demo of `alldeletedrevisions` module: List the all deleted revisions from User",
+        "filename": "filearchive",
+        "docstring": "Enumerate 3 deleted files from filearchive table in descending order.",
         "endpoint": "https://en.wikipedia.org/w/api.php",
         "params": {
             "action": "query",
-            "list": "alldeletedrevisions",
-            "adruser": "Mahesh",
-			"adrprop": "ids|user|comment",
-			"format": "json"
+			"format": "json",
```

Comment:
The indentation of the schema is inconsistent. Check if you are mixing tabs and spaces.